contracts/ics-escrow/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2022 callumanderson <callumanderson745@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
Not sure we need these notices along with the actual licenses... seems to be another thing from `cw-template` code gen.

Reply:
Agree. Think we should be OK with a licence in the root. Removed.